pulsar-broker/src/main/java/org/apache/pulsar/broker/PulsarService.java

```diff
@@ -734,8 +734,17 @@ public void start() throws PulsarServerException {
             schemaRegistryService = SchemaRegistryService.create(
                     schemaStorage, config.getSchemaRegistryCompatibilityCheckers(), this.executor);
 
-            this.defaultOffloader = createManagedLedgerOffloader(
-                    OffloadPoliciesImpl.create(this.getConfiguration().getProperties()));
+            OffloadPoliciesImpl defaultOffloadPolicies =
+                    OffloadPoliciesImpl.create(this.getConfiguration().getProperties());
+            this.defaultOffloader = createManagedLedgerOffloader(defaultOffloadPolicies);
```

Comment:
We should make 
```
 this.defaultOffloader = createManagedLedgerOffloader(defaultOffloadPolicies);
```
after 
```
offloaderStats = LedgerOffloaderStats.create(config.isExposeManagedLedgerMetricsInPrometheus(),
      exposeTopicMetrics, offloaderScheduler, interval);
```

In `createManagedLedgerOffloader`, it will use `offloaderStats`. We shuold make this method after `offloaderStats` override.